src/main/java/com/hms_networks/americas/sc/extensions/historicaldata/HistoricalDataConstants.java

```diff
@@ -55,4 +55,10 @@
 
   /** Minimum amount of Historical FIFO queue that can be set by library users. */
   static final long MIN_QUEUE_SPAN_MINS = 1;
+
+  /** The number of milliseconds in one hour. */
+  static final long MILLISECONDS_PER_HOUR = 3600000L;
+
+  /** Default maximum amount of time Historical FIFO can get behind: 48 hours in ms. */
+  static final long MAX_HIST_FIFO_GET_BEHIND_MS = 48L * MILLISECONDS_PER_HOUR;
```

Comment:
```suggestion
  /** Default maximum amount of time Historical FIFO can get behind: 48 hours in ms. */
  static final long MAX_HIST_FIFO_GET_BEHIND_MS = SCTimeUnit.HOURS.toMillis(48);
```

Reply:
If this is done you can remove the "48 hours in ms." in the comment as the code documents this well.

Reply:
changed.